addons/google_account/google_account.py

```diff
@@ -89,14 +89,17 @@ def _get_google_token_json(self, cr, uid, authorize_code, service, context=None)
 
         headers = {"content-type": "application/x-www-form-urlencoded"}
 
-        try:
-            uri = self.get_uri_oauth(a='token')
-            data = werkzeug.url_encode(params)
+        if client_id and client_secret:
```

Comment:
Yeah, that's what I mean. 

As there's a `return` statement, you don't need to put `else` sentence. It's a bit less readable (but most IDEs highlight `return`, so not at all), but you prevent to raise cyclomatic complexity (https://en.wikipedia.org/wiki/Cyclomatic_complexity)